src/operations/claim_claimable_balance.js

```diff
@@ -0,0 +1,33 @@
+import xdr from '../generated/stellar-xdr_generated';
+
+/**
+ * Create a new claim claimable balance operation.
+ * @function
+ * @alias Operation.claimClaimableBalance
+ * @param {object} opts Options object
+ * @param {string} opts.claimableBalanceId - The claimable balance id to be claimed.
+ * @param {string} [opts.source] - The source account for the operation. Defaults to the transaction's source account.
+ * @returns {xdr.Operation} Claim claimable balance operation
+ *
+ * @example
+ * const op = Operation.claimClaimableBalance({
+ *   balanceId: '00000000da0d57da7d4850e7fc10d2a9d0ebc731f7afb40574c03395b17d49149b91f5be',
+ * });
+ *
+ */
+export function claimClaimableBalance(opts) {
+  if (typeof opts.balanceId !== 'string' || opts.balanceId.length < 2) {
```

Comment:
Wait, why length < 2? 

Also, the docstring above says `opts.claimableBalanceId` which doesn't match `opts.balanceId` here.

Reply:
good catch! Actually we don't need the length check, we can delegate that to the XDR library.